content/php/concepts/math-functions/terms/cosh/cosh.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'cosh()'
+Description: 'Returns the hyperbolic cosine of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`cosh()`** function in PHP returns the hyperbolic cosine of a number. It calculates the value using the exponential function and is useful in various mathematical calculations.
+
+## Syntax
+
+```pseudo
+$result = cosh(number);
+```
+
+`number` - represents the input value for which the hyperbolic cosine will be returned.
+
+## Example
+
+In the following example the `cosh()` is used to return the hyperbolic cosine of `2.5`:
+
+```php
+<?php
+
+  echo(cosh(2.5));
+
+?>
+```
+
+The output will be:
+
+```shell
+6.1322894796637
+```
+
+## Codebyte Example
```

Comment:
We don't duplicate examples in docs. I would recommend deleting the codebyte unless there is some additional functionality that is worthwhile illustrating with a different example.